src/test/java/guitests/GuiRobot.java

```diff
@@ -15,7 +15,7 @@
 public class GuiRobot extends FxRobot {
 
     private static final int PAUSE_FOR_HUMAN_DELAY_MILLISECONDS = 250;
-    private static final int DEFAULT_WAIT_FOR_EVENT_TIMEOUT_MILLISECONDS = 5000;
+    private static final int DEFAULT_WAIT_FOR_EVENT_TIMEOUT_MILLISECONDS = 50000;
```

Comment:
Just curious, what's the reason for extending this timeout? 🤔

Reply:
My internet sucks :(